<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MONAIBundle</class>
 <widget class="qMRMLWidget" name="MONAIBundle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>606</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Input volume :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector" native="true">
        <property name="toolTip">
         <string>Input abdominal, chest, or whole body CT.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="remoteBundleCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Select remote bundle</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="updateModelsFromZooButton">
        <property name="text">
         <string>Update models from Monai Zoo</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="selectModelLabel">
          <property name="text">
           <string>Select model:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="modelZooSelectionBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="selectModelVersionLabel">
          <property name="text">
           <string>Select model version:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="modelVersionSelectionBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="bundleStorageDirLabel">
          <property name="text">
           <string>select storage dir for bundles:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ctkPathLineEdit" name="bundleStorageDirSelector"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="downloadModelFromZooButton">
        <property name="text">
         <string>Download Model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="localBundleCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Select local bundle</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="bundleRootDirLabel">
        <property name="text">
         <string>Bundle root folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="bundleRootDirSelector"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="bundleInfoLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="bundleDescriptionLabel">
       <property name="text">
        <string>Loaded bundle:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPlainTextEdit" name="bundleDescriptionTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>No bundle loaded</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="deviceSelectionLabel">
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="deviceSelectionBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cpu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cuda</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Start segmentation.</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ctkpathlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
